src/resolvers-cassandra/Messages/queries.js

```diff
@@ -119,46 +120,47 @@ function byLocation(args, res) { // eslint-disable-line no-unused-vars
 }
 
 /**
- * @param {site: string, originalSource: string, bbox: number[], mainTerm: string, filteredEdges: string[], langCode: string, limit: number, offset: number, fromDate: string, toDate: string, sourceFilter: string[], fulltextTerm: string} args
+ * @param {externalsourceid: string, bbox: number[], conjunctivetopics: string[], limit: number, pageState: number, fromDate: string, toDate: string, pipelinekeys: string[], fulltextTerm: string} args
  * @returns {Promise.<{runTime: string, type: string, bbox: number[], features: Feature[]}>}
- */
+ **/
 function byBbox(args, res) { // eslint-disable-line no-unused-vars
   return new Promise((resolve, reject) => {
     if (!args.bbox || args.bbox.length !== 4) return reject('Invalid bbox specified');
+    if (!args.conjunctivetopics || args.conjunctivetopics.length === 0) return reject('Empty conjunctive topic list specified');
```

Comment:
This is safer as `!args.conjunctivetopics.length` which catches the case where conjunctivetopics is empty but also when it's not an array.